### PR TITLE
ADPT-147: Added domain parameter to the creation of new tasks

### PR DESCRIPTION
**Pull request by holgerhagen**
---
 
### For the submitter:
- [x] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [x] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

**Comment by mustaphazorgati** [comment]
Is there a reason why the ticket is not a `TSK` ticket?
This will break taskana-release-script.

**Comment by mustaphazorgati** [comment]
I just checked: our commit-msg hook allows ADPT ticket for this repository. This should not be allowed

File: lib/taskana-core/src/test/resources/taskana.properties [hunk 1]
```diff
@@ -1,20 +1,64 @@
...
 taskana.roles.taskadmin=taskadmin
+
+#################################################
+# TASKANA domain configurations
+#################################################
+
+# The comma separated list of administrative domains
 taskana.domains=Domain_A  ,  DOMAIN_B
+# The default domain to create new tasks in if no other domain is specified
+taskana.domains.default=DOMAIN_A
```
Comment (gitgoodjhe) [inline lib/taskana-core/src/test/resources/taskana.properties:23]: Should we also add this to the properties of the example?

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

File: lib/taskana-core/src/test/resources/taskana.properties (hunk 1 above)
Comment (holgerhagen) [reply lib/taskana-core/src/test/resources/taskana.properties:23]: ok.

**Review by gitgoodjhe — APPROVED** [review]

**Pull request closed without merging**